src/library/baseplaylistfeature.cpp

```diff
@@ -402,18 +410,18 @@ void BasePlaylistFeature::slotCreateImportPlaylist() {
     }
 
     // Set last import directory
-    QFileInfo fileName(playlist_files.first());
-    m_pConfig->set(ConfigKey("[Library]", "LastImportExportPlaylistDirectory"),
-            ConfigValue(fileName.dir().absolutePath()));
+    QFileInfo fileInfo(playlist_files.first());
+    m_pConfig->set(kConfigKeyLastImportExportPlaylistDirectory,
+            ConfigValue(fileInfo.dir().absolutePath()));
 
     int lastPlaylistId = -1;
 
     // For each selected element create a different playlist.
     for (const QString& playlistFile : playlist_files) {
-        fileName = QFileInfo(playlistFile);
+        fileInfo = QFileInfo(playlistFile);
 
         // Get a valid name
-        QString baseName = fileName.baseName();
+        QString baseName = fileInfo.baseName();
         QString name;
```

Comment:
if we keep fileInfo from above, what makes more sense:
reuse it here, or instantiate a new one (const) here for the sake of const?

Reply:
Well we're already instantiating a new anyways and just assigning the new one to the variable of the old one (the compiler can't know at compile time that they're the same object). The only possible result is that the resulting function might create another local variable on the stack, but space on the stack is dirt-cheap so that doesn't really matter IMO.